### ISSUE TITLE
[Device Support Request] Aqara E1 door/window sensor MCCGQ14LM

### ISSUE THREAD

**Issue by haralake** [zigpy/zha-device-handlers#1088]
**Is your feature request related to a problem? Please describe.**
I'm using Home assistant version 2021.10.6 with conbee 2 and  i'm managed to pair the sensor with success. The problem is that the sensor is not working properly.

**Describe the solution you'd like**
Device is not supported

**Device signature - this can be acquired by removing the device from ZHA and pairing it again from the add devices screen. Be sure to add the entire content of the log panel after pairing the device to a code block below this line.**
```
{
  "node_descriptor": "NodeDescriptor(logical_type=<LogicalType.EndDevice: 2>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress: 128>, manufacturer_code=4447, maximum_buffer_size=66, maximum_incoming_transfer_size=66, server_mask=10752, maximum_outgoing_transfer_size=66, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=True, *is_full_function_device=False, *is_mains_powered=False, *is_receiver_on_when_idle=False, *is_router=False, *is_security_capable=False)",
  "endpoints": {
    "1": {
      "profile_id": 260,
      "device_type": "0x0402",
      "in_clusters": [
        "0x0000",
        "0x0001",
        "0x0003",
        "0x0500",
        "0xfcc0"
      ],
      "out_clusters": [
        "0x0003",
        "0x0019"
      ]
    }
  },
  "manufacturer": "LUMI",
  "model": "lumi.magnet.acn001",
  "class": "zigpy.device.Device"
}
```

**Additional context**
![Καταγραφή στο Web_21-10-2021_14316_192 168 1 20](https://user-images.githubusercontent.com/18325407/138265094-b8548540-cffc-460e-989c-5c5e7894f63f.jpeg)

**Comment by MattWestb** [comment]
Its one new device that is using isa zone and the old was using the normal on/off cluster.
```
    cluster_id = 0x0500
    name = "IAS Zone"
    ep_attribute = "ias_zone"
```
https://github.com/zigpy/zigpy/blob/f0f12e69043ac05252ac50dbc26a4ade4ff9c62d/zigpy/zcl/clusters/security.py#L62-L64 
Some need revers engineering the setting for the device and implanting it in one quirk for the device ;-(

Or perhaps need looking how the ISA zone is getting the command or not.

**Comment by Adminiuga** [comment]
and the hope of E1 devices being normal Zigbee devices quickly goes out of the window

**Comment by MattWestb** [comment]
Looks like ZHA is doing no config then its one "lumi" device and its very polite and reporting one attribute 9 times `Attribute report received: model=lumi.magnet.acn001`

**Comment by Adminiuga** [comment]
```
2021-10-21 14:46:57 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x647B:1:0x0500]: bound 'ias_zone' cluster: Status.TABLE_FULL
```

I guess we do bind the IAS cluster, but ran out of space.

```
2021-10-21 14:46:58 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x647B:1:0x0500]: wrote cie_addr: 84:2e:14:ff:fe:ad:2c:d4 to 'ias_zone' cluster: [WriteAttributesStatusRecord(status=<Status.SUCCESS: 0>)]
2021-10-21 14:46:58 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x647B:1:0x0500]: Sending pro-active IAS enroll response
```

We did write the CIEEE address successfully and sent it an enroll response.

But no other traffic on 0x0500 cluster 🤷

**Comment by BuggeringHell** [comment]
Looks like it is due to be added to the deconz plugin, if that is of any help to the ZHA side of development
#https://github.com/dresden-elektronik/deconz-rest-plugin/pull/5434/commits/010938bf929362b535b0cf420e1e60796cf4f1f3

**Comment by TomLyon** [comment]
I just received a huge lot of these today and they are all showing up as null devices. Hoping these can be supported.

**Comment by Adminiuga** [comment]
What do you mean: null devices

**Comment by TomLyon** [comment]
Nothing different than what's already been said just that they're perpetually stuck in the off state and opening/closing does nothing. I've added my device join logs below. If there's anything else I can provide I will do my best to provide it. 

`[0x0000:zdo] ZDO request ZDOCmd.Mgmt_Permit_Joining_req: [60, <Bool.false: 0>]
New device 0x9916 (54:ef:44:10:00:25:82:ef) joined the network
[0x9916] Scheduling initialization
Tries remaining: 3
[0x9916] Requesting 'Node Descriptor'
Tries remaining: 2
[0x9916] Extending timeout for 0xce request
Device 0x9916 (54:ef:44:10:00:25:82:ef) joined the network
[0x9916] Scheduling initialization
[0x9916] Canceling old initialize call
Received frame on uninitialized device <Device model=None manuf=None nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=False> from ep 0 to ep 0, cluster 19: b'\x00\x16\x99\xef\x82%\x00\x10D\xefT\x80'
[0x9916:zdo] ZDO request ZDOCmd.Device_annce: [0x9916, 54:ef:44:10:00:25:82:ef, 128]
Received frame on uninitialized device <Device model=None manuf=None nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=False> from ep 0 to ep 0, cluster 32770: b'\xce\x00\x16\x99\x02@\x80_\x11BB\x00\x00*B\x00\x00'
Tries remaining: 3
[0x9916] Requesting 'Node Descriptor'
Tries remaining: 2
[0x9916] Extending timeout for 0xd0 request
[0x9916] Got Node Descriptor: NodeDescriptor(logical_type=<LogicalType.EndDevice: 2>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress: 128>, manufacturer_code=4447, maximum_buffer_size=66, maximum_incoming_transfer_size=66, server_mask=10752, maximum_outgoing_transfer_size=66, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=True, *is_full_function_device=False, *is_mains_powered=False, *is_receiver_on_when_idle=False, *is_router=False, *is_security_capable=False)
[0x9916] Discovering endpoints
Tries remaining: 3
[0x9916] Extending timeout for 0xd2 request
Received frame on uninitialized device <Device model=None manuf=None nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=False> from ep 0 to ep 0, cluster 32770: b'\xd0\x00\x16\x99\x02@\x80_\x11BB\x00\x00*B\x00\x00'
[0x9916] Got Node Descriptor: NodeDescriptor(logical_type=<LogicalType.EndDevice: 2>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress: 128>, manufacturer_code=4447, maximum_buffer_size=66, maximum_incoming_transfer_size=66, server_mask=10752, maximum_outgoing_transfer_size=66, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=True, *is_full_function_device=False, *is_mains_powered=False, *is_receiver_on_when_idle=False, *is_router=False, *is_security_capable=False)
[0x9916] Discovering endpoints
Tries remaining: 3
[0x9916] Extending timeout for 0xd4 request
Received frame on uninitialized device <Device model=None manuf=None nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=False> from ep 0 to ep 0, cluster 32770: b'\xd0\x00\x16\x99\x02@\x80_\x11BB\x00\x00*B\x00\x00'
[0x9916:zdo] ZDO request ZDOCmd.Node_Desc_rsp: [<Status.SUCCESS: 0>, 0x9916, NodeDescriptor(logical_type=<LogicalType.EndDevice: 2>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.AllocateAddress: 128>, manufacturer_code=4447, maximum_buffer_size=66, maximum_incoming_transfer_size=66, server_mask=10752, maximum_outgoing_transfer_size=66, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=True, *is_full_function_device=False, *is_mains_powered=False, *is_receiver_on_when_idle=False, *is_router=False, *is_security_capable=False)]
[0x9916:zdo] Unsupported ZDO request:ZDOCmd.Node_Desc_rsp
Received frame on uninitialized device <Device model=None manuf=None nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=False> from ep 0 to ep 0, cluster 32773: b'\xd4\x00\x16\x99\x01\x01'
[0x9916] Discovered endpoints: [1]
[0x9916] Initializing endpoints [<Endpoint id=1 in=[] out=[] status=<Status.NEW: 0>>]
[0x9916:1] Discovering endpoint information
Tries remaining: 3
[0x9916] Extending timeout for 0xd6 request
Received frame on uninitialized device <Device model=None manuf=None nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=False> from ep 0 to ep 0, cluster 32772: b'\xd6\x00\x16\x99\x16\x01\x04\x01\x02\x04\x02\x05\x00\x00\x01\x00\x03\x00\x00\x05\xc0\xfc\x02\x03\x00\x19\x00'
[0x9916:1] Discovered endpoint information: SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=1026, device_version=2, input_clusters=[0, 1, 3, 1280, 64704], output_clusters=[3, 25])
[0x9916] Extending timeout for 0xd8 request
[0x9916:1:0x0000] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=133 command_id=Command.Report_Attributes>
[0x9916:1:0x0000] ZCL request 0x000a: [[Attribute(attrid=1, value=<TypeValue type=uint8_t, value=4>), Attribute(attrid=5, value=<TypeValue type=CharacterString, value=lumi.magnet.acn001>)]]
[0x9916:1:0x0000] Attribute report received: app_version=4, model=lumi.magnet.acn001
Error handling '_save_attribute' event with (54:ef:44:10:00:25:82:ef, 1, 0, 1, 4) params: FOREIGN KEY constraint failed
Error handling '_save_attribute' event with (54:ef:44:10:00:25:82:ef, 1, 0, 5, 'lumi.magnet.acn001') params: FOREIGN KEY constraint failed
[0x9916:1:0x0000] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=216 command_id=Command.Read_Attributes_rsp>
[0x9916] Read model 'lumi.magnet.acn001' and manufacturer 'LUMI' from <Endpoint id=1 in=[basic:0x0000, power:0x0001, identify:0x0003, ias_zone:0x0500, manufacturer_specific:0xFCC0] out=[identify:0x0003, ota:0x0019] status=<Status.ZDO_INIT: 1>>
[0x9916] Discovered basic device information for <Device model='lumi.magnet.acn001' manuf='LUMI' nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=True>
Device is initialized <Device model='lumi.magnet.acn001' manuf='LUMI' nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=True>
Checking quirks for LUMI lumi.magnet.acn001 (54:ef:44:10:00:25:82:ef)
Considering <class 'zhaquirks.xbee.xbee_io.XBeeSensor'>
Fail because endpoint list mismatch: {232, 230} {1}
Considering <class 'zhaquirks.xbee.xbee3_io.XBee3Sensor'>
Fail because endpoint list mismatch: {232, 230} {1}
Considering <class 'zhaquirks.smartthings.tag_v4.SmartThingsTagV4'>
Fail because device_type mismatch on at least one endpoint
Considering <class 'zhaquirks.smartthings.multi.SmartthingsMultiPurposeSensor'>
Fail because input cluster mismatch on at least one endpoint
Considering <class 'zhaquirks.netvox.z308e3ed.Z308E3ED'>
Fail because input cluster mismatch on at least one endpoint
Considering <class 'zhaquirks.gledopto.soposhgu10.SoposhGU10'>
Fail because endpoint list mismatch: {11, 13} {1}
Considering <class 'bellows.zigbee.application.EZSPCoordinator'>
Fail because device_type mismatch on at least one endpoint
device - 0x9916:54:ef:44:10:00:25:82:ef entering async_device_initialized - is_new_join: True
device - 0x9916:54:ef:44:10:00:25:82:ef has joined the ZHA zigbee network
[0x9916](lumi.magnet.acn001): started configuration
[0x9916:ZDO](lumi.magnet.acn001): 'async_configure' stage succeeded
[0x9916] Extending timeout for 0xda request
[0x9916] Extending timeout for 0xdc request
[0x9916:1:0x0000]: finished channel configuration
[0x9916] Extending timeout for 0xde request
Error handling '_save_attribute' event with (54:ef:44:10:00:25:82:ef, 1, 0, 4, 'LUMI') params: FOREIGN KEY constraint failed
Error handling '_save_attribute' event with (54:ef:44:10:00:25:82:ef, 1, 0, 5, 'lumi.magnet.acn001') params: FOREIGN KEY constraint failed
[0x9916:1:0x0001]: bound 'power' cluster: Status.SUCCESS
[0x9916] Extending timeout for 0xe0 request
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=220 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0500]: started IASZoneChannel configuration
[0x9916] Extending timeout for 0xe2 request
[0x9916:1:0xfcc0]: bound 'manufacturer_specific' cluster: Status.SUCCESS
[0x9916:1:0xfcc0]: finished channel configuration
[0x9916:1:0x0019]: finished channel configuration
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=224 command_id=Command.Configure_Reporting_rsp>
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=224 command_id=Command.Configure_Reporting_rsp>
[0x9916] Invalid state on future for 0xe0 seq -- probably duplicate response
[0x9916:1:0x0500]: bound 'ias_zone' cluster: Status.TABLE_FULL
[0x9916] Extending timeout for 0xe4 request
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=228 command_id=Command.Write_Attributes_rsp>
[0x9916:1:0x0500]: wrote cie_addr: 00:0d:6f:00:17:21:88:f6 to 'ias_zone' cluster: [WriteAttributesStatusRecord(status=<Status.SUCCESS: 0>)]
[0x9916:1:0x0500]: Sending pro-active IAS enroll response
[0x9916:1:0x0500]: finished IASZoneChannel configuration
[0x9916] Extending timeout for 0xe6 request
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=230 command_id=Command.Default_Response>
[0x9916] Discovered endpoints: [1]
[0x9916] Initializing endpoints [<Endpoint id=1 in=[] out=[] status=<Status.NEW: 0>>]
[0x9916:1] Discovering endpoint information
Tries remaining: 3
[0x9916] Extending timeout for 0xe8 request
Received frame on uninitialized device <Device model='lumi.magnet.acn001' manuf='LUMI' nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=False> from ep 0 to ep 0, cluster 32772: b'\xe8\x00\x16\x99\x16\x01\x04\x01\x02\x04\x02\x05\x00\x00\x01\x00\x03\x00\x00\x05\xc0\xfc\x02\x03\x00\x19\x00'
[0x9916] Scheduling initialization
Tries remaining: 3
[0x9916] Already have endpoints: {0: <zigpy.zdo.ZDO object at 0x7f7cead474c0>, 1: <Endpoint id=1 in=[] out=[] status=<Status.NEW: 0>>}
[0x9916] Initializing endpoints [<Endpoint id=1 in=[] out=[] status=<Status.NEW: 0>>]
[0x9916:1] Discovering endpoint information
Tries remaining: 3
[0x9916] Extending timeout for 0xea request
[0x9916:1] Discovered endpoint information: SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=1026, device_version=2, input_clusters=[0, 1, 3, 1280, 64704], output_clusters=[3, 25])
[0x9916] Already have model and manufacturer info
[0x9916] Discovered basic device information for <Device model='lumi.magnet.acn001' manuf='LUMI' nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=True>
Device is initialized <Device model='lumi.magnet.acn001' manuf='LUMI' nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=True>
Checking quirks for LUMI lumi.magnet.acn001 (54:ef:44:10:00:25:82:ef)
Considering <class 'zhaquirks.xbee.xbee_io.XBeeSensor'>
Fail because endpoint list mismatch: {232, 230} {1}
Considering <class 'zhaquirks.xbee.xbee3_io.XBee3Sensor'>
Fail because endpoint list mismatch: {232, 230} {1}
Considering <class 'zhaquirks.smartthings.tag_v4.SmartThingsTagV4'>
Fail because device_type mismatch on at least one endpoint
Considering <class 'zhaquirks.smartthings.multi.SmartthingsMultiPurposeSensor'>
Fail because input cluster mismatch on at least one endpoint
Considering <class 'zhaquirks.netvox.z308e3ed.Z308E3ED'>
Fail because input cluster mismatch on at least one endpoint
Considering <class 'zhaquirks.gledopto.soposhgu10.SoposhGU10'>
Fail because endpoint list mismatch: {11, 13} {1}
Considering <class 'bellows.zigbee.application.EZSPCoordinator'>
Fail because device_type mismatch on at least one endpoint
device - 0x9916:54:ef:44:10:00:25:82:ef entering async_device_initialized - is_new_join: True
device - 0x9916:54:ef:44:10:00:25:82:ef has joined the ZHA zigbee network
[0x9916](lumi.magnet.acn001): started configuration
[0x9916:ZDO](lumi.magnet.acn001): 'async_configure' stage succeeded
[0x9916] Extending timeout for 0xec request
[0x9916] Extending timeout for 0xee request
Device 54:ef:44:10:00:25:82:ef already exists. Updating it.
[0x9916:1] Discovered endpoint information: SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=1026, device_version=2, input_clusters=[0, 1, 3, 1280, 64704], output_clusters=[3, 25])
[0x9916] Already have model and manufacturer info
[0x9916] Discovered basic device information for <Device model='lumi.magnet.acn001' manuf='LUMI' nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=True>
Device is initialized <Device model='lumi.magnet.acn001' manuf='LUMI' nwk=0x9916 ieee=54:ef:44:10:00:25:82:ef is_initialized=True>
Checking quirks for LUMI lumi.magnet.acn001 (54:ef:44:10:00:25:82:ef)
Considering <class 'zhaquirks.xbee.xbee_io.XBeeSensor'>
Fail because endpoint list mismatch: {232, 230} {1}
Considering <class 'zhaquirks.xbee.xbee3_io.XBee3Sensor'>
Fail because endpoint list mismatch: {232, 230} {1}
Considering <class 'zhaquirks.smartthings.tag_v4.SmartThingsTagV4'>
Fail because device_type mismatch on at least one endpoint
Considering <class 'zhaquirks.smartthings.multi.SmartthingsMultiPurposeSensor'>
Fail because input cluster mismatch on at least one endpoint
Considering <class 'zhaquirks.netvox.z308e3ed.Z308E3ED'>
Fail because input cluster mismatch on at least one endpoint
Considering <class 'zhaquirks.gledopto.soposhgu10.SoposhGU10'>
Fail because endpoint list mismatch: {11, 13} {1}
Considering <class 'bellows.zigbee.application.EZSPCoordinator'>
Fail because device_type mismatch on at least one endpoint
device - 0x9916:54:ef:44:10:00:25:82:ef entering async_device_initialized - is_new_join: True
device - 0x9916:54:ef:44:10:00:25:82:ef has joined the ZHA zigbee network
[0x9916](lumi.magnet.acn001): started configuration
[0x9916:ZDO](lumi.magnet.acn001): 'async_configure' stage succeeded
Device 54:ef:44:10:00:25:82:ef already exists. Updating it.
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=238 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0001]: bound 'power' cluster: Status.SUCCESS
[0x9916] Extending timeout for 0xf0 request
[0x9916:1:0x0500]: started IASZoneChannel configuration
[0x9916] Extending timeout for 0xf2 request
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=240 command_id=Command.Configure_Reporting_rsp>
[0x9916:1:0x0000]: finished channel configuration
[0x9916] Extending timeout for 0xf4 request
[0x9916:1:0x0500]: bound 'ias_zone' cluster: Status.TABLE_FULL
[0x9916] Extending timeout for 0xf6 request
[0x9916:1:0xfcc0]: bound 'manufacturer_specific' cluster: Status.SUCCESS
[0x9916:1:0xfcc0]: finished channel configuration
[0x9916:1:0x0019]: finished channel configuration
[0x9916] Extending timeout for 0xf8 request
[0x9916] Extending timeout for 0xfa request
[0x9916:1:0x0001]: 'async_configure' stage failed: 'ConfigureReportingResponse' object has no attribute 'status'
[0x9916:1:0x0500]: 'async_configure' stage succeeded
[0x9916:1:0x0000]: 'async_configure' stage succeeded
[0x9916:1:0xfcc0]: 'async_configure' stage succeeded
[0x9916:1:0x0019]: 'async_configure' stage succeeded
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=224 command_id=Command.Configure_Reporting_rsp>
[0x9916:1:0x0001] ZCL request 0x0007: [[ConfigureReportingResponseRecord(status=134, direction=0, attrid=32), ConfigureReportingResponseRecord(status=134, direction=0, attrid=33)]]
[0x9916](lumi.magnet.acn001): completed configuration
[0x9916](lumi.magnet.acn001): stored in registry: ZhaDeviceEntry(name='LUMI lumi.magnet.acn001', ieee='54:ef:44:10:00:25:82:ef', last_seen=1635817602.3984718)
[0x9916] Extending timeout for 0xfc request
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=246 command_id=Command.Write_Attributes_rsp>
[0x9916:1:0x0500]: wrote cie_addr: 00:0d:6f:00:17:21:88:f6 to 'ias_zone' cluster: [WriteAttributesStatusRecord(status=<Status.SUCCESS: 0>)]
[0x9916:1:0x0500]: Sending pro-active IAS enroll response
[0x9916:1:0x0500]: finished IASZoneChannel configuration
[0x9916] Extending timeout for 0xfe request
[0x9916:1:0x0000]: finished channel configuration
[0x9916] Extending timeout for 0x00 request
[0x9916:1:0x0001]: 'async_configure' stage failed: 'ConfigureReportingResponse' object has no attribute 'status'
[0x9916:1:0x0500]: 'async_configure' stage succeeded
[0x9916:1:0x0000]: 'async_configure' stage succeeded
[0x9916:1:0xfcc0]: 'async_configure' stage succeeded
[0x9916:1:0x0019]: 'async_configure' stage succeeded
[0x9916](lumi.magnet.acn001): completed configuration
[0x9916](lumi.magnet.acn001): stored in registry: ZhaDeviceEntry(name='LUMI lumi.magnet.acn001', ieee='54:ef:44:10:00:25:82:ef', last_seen=1635817602.4320514)
[0x9916] Extending timeout for 0x02 request
[0x9916:1:0x0001]: bound 'power' cluster: Status.SUCCESS
[0x9916] Extending timeout for 0x04 request
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=250 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0500]: started IASZoneChannel configuration
[0x9916] Extending timeout for 0x06 request
[0x9916:1:0x0003] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=252 command_id=Command.Default_Response>
[0x9916:1:0x0003]: executed 'trigger_effect' command with args: '(2, 0)' kwargs: '{}' result: [64, <Status.SUCCESS: 0>]
[0x9916](lumi.magnet.acn001): started initialization
[0x9916:ZDO](lumi.magnet.acn001): 'async_initialize' stage succeeded
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=254 command_id=Command.Default_Response>
[0x9916:1:0xfcc0]: bound 'manufacturer_specific' cluster: Status.SUCCESS
[0x9916:1:0xfcc0]: finished channel configuration
[0x9916:1:0x0019]: finished channel configuration
[0x9916:1:0x0001]: initializing channel: from_cache: False
[0x9916] Extending timeout for 0x08 request
[0x9916:1:0x0003] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=2 command_id=Command.Default_Response>
[0x9916:1:0x0003]: executed 'trigger_effect' command with args: '(2, 0)' kwargs: '{}' result: [64, <Status.SUCCESS: 0>]
[0x9916](lumi.magnet.acn001): started initialization
[0x9916:ZDO](lumi.magnet.acn001): 'async_initialize' stage succeeded
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=4 command_id=Command.Configure_Reporting_rsp>
[0x9916:1:0x0500]: initializing channel: from_cache: False
[0x9916] Extending timeout for 0x0a request
[0x9916:1:0x0500]: bound 'ias_zone' cluster: Status.TABLE_FULL
[0x9916] Extending timeout for 0x0c request
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=8 command_id=Command.Read_Attributes_rsp>
[0x9916] Extending timeout for 0x0e request
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=10 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0500]: finished channel configuration
[0x9916:1:0x0000]: initializing channel: from_cache: False
[0x9916:1:0x0000]: finished channel configuration
[0x9916:1:0xfcc0]: initializing channel: from_cache: False
[0x9916:1:0xfcc0]: finished channel configuration
[0x9916:1:0x0019]: initializing channel: from_cache: False
[0x9916:1:0x0019]: finished channel configuration
[0x9916:1:0x0001]: initializing channel: from_cache: False
[0x9916] Extending timeout for 0x10 request
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=12 command_id=Command.Write_Attributes_rsp>
[0x9916:1:0x0500]: wrote cie_addr: 00:0d:6f:00:17:21:88:f6 to 'ias_zone' cluster: [WriteAttributesStatusRecord(status=<Status.SUCCESS: 0>)]
[0x9916:1:0x0500]: Sending pro-active IAS enroll response
[0x9916:1:0x0500]: finished IASZoneChannel configuration
[0x9916] Extending timeout for 0x12 request
[0x9916:1:0x0500]: initializing channel: from_cache: False
[0x9916] Extending timeout for 0x14 request
[0x9916:1:0x0001]: 'async_configure' stage failed: 'ConfigureReportingResponse' object has no attribute 'status'
[0x9916:1:0x0500]: 'async_configure' stage succeeded
[0x9916:1:0x0000]: 'async_configure' stage succeeded
[0x9916:1:0xfcc0]: 'async_configure' stage succeeded
[0x9916:1:0x0019]: 'async_configure' stage succeeded
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=14 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=16 command_id=Command.Read_Attributes_rsp>
[0x9916](lumi.magnet.acn001): completed configuration
[0x9916](lumi.magnet.acn001): stored in registry: ZhaDeviceEntry(name='LUMI lumi.magnet.acn001', ieee='54:ef:44:10:00:25:82:ef', last_seen=1635817604.4410174)
[0x9916] Extending timeout for 0x16 request
[0x9916:1:0x0001]: finished channel configuration
[0x9916] Extending timeout for 0x18 request
[0x9916:1:0x0000]: initializing channel: from_cache: False
[0x9916:1:0x0000]: finished channel configuration
[0x9916:1:0xfcc0]: initializing channel: from_cache: False
[0x9916:1:0xfcc0]: finished channel configuration
[0x9916:1:0x0001]: 'async_initialize' stage succeeded
[0x9916:1:0x0500]: 'async_initialize' stage succeeded
[0x9916:1:0x0000]: 'async_initialize' stage succeeded
[0x9916:1:0xfcc0]: 'async_initialize' stage succeeded
[0x9916:1:0x0019]: 'async_initialize' stage succeeded
[0x9916:1:0x0019]: initializing channel: from_cache: False
[0x9916:1:0x0019]: finished channel configuration
[0x9916](lumi.magnet.acn001): power source: Battery or Unknown
[0x9916](lumi.magnet.acn001): completed initialization
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=18 command_id=Command.Default_Response>
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=20 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0500]: finished channel configuration
[0x9916:1:0x0003] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=22 command_id=Command.Default_Response>
[0x9916:1:0x0003]: executed 'trigger_effect' command with args: '(2, 0)' kwargs: '{}' result: [64, <Status.SUCCESS: 0>]
[0x9916](lumi.magnet.acn001): started initialization
[0x9916:ZDO](lumi.magnet.acn001): 'async_initialize' stage succeeded
[0x9916:1:0x0001]: initializing channel: from_cache: False
[0x9916] Extending timeout for 0x1a request
[0x9916:1:0x0500]: initializing channel: from_cache: False
[0x9916] Extending timeout for 0x1c request
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=24 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0001]: finished channel configuration
[0x9916:1:0x0000]: initializing channel: from_cache: False
[0x9916:1:0x0000]: finished channel configuration
[0x9916:1:0xfcc0]: initializing channel: from_cache: False
[0x9916:1:0xfcc0]: finished channel configuration
[0x9916:1:0x0001]: 'async_initialize' stage succeeded
[0x9916:1:0x0500]: 'async_initialize' stage succeeded
[0x9916:1:0x0000]: 'async_initialize' stage succeeded
[0x9916:1:0xfcc0]: 'async_initialize' stage succeeded
[0x9916:1:0x0019]: 'async_initialize' stage succeeded
[0x9916:1:0x0019]: initializing channel: from_cache: False
[0x9916:1:0x0019]: finished channel configuration
[0x9916](lumi.magnet.acn001): power source: Battery or Unknown
[0x9916](lumi.magnet.acn001): completed initialization
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=26 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=28 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0500]: finished channel configuration
[0x9916] Extending timeout for 0x1e request
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=26 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0001] ZCL request 0x0001: [[ReadAttributeRecord(attrid=32, status=<Status.SUCCESS: 0>, value=<TypeValue type=uint8_t, value=28>), ReadAttributeRecord(attrid=33, status=<Status.UNSUPPORTED_ATTRIBUTE: 134>)]]
[0x9916:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=True disable_default_response=True> manufacturer=None tsn=30 command_id=Command.Read_Attributes_rsp>
[0x9916:1:0x0001]: finished channel configuration
[0x9916:1:0x0001]: 'async_initialize' stage succeeded
[0x9916:1:0x0500]: 'async_initialize' stage succeeded
[0x9916:1:0x0000]: 'async_initialize' stage succeeded
[0x9916:1:0xfcc0]: 'async_initialize' stage succeeded
[0x9916:1:0x0019]: 'async_initialize' stage succeeded
[0x9916](lumi.magnet.acn001): power source: Battery or Unknown
[0x9916](lumi.magnet.acn001): completed initialization`

**Comment by ketixar** [comment]
Looks like Deconz and Z2M managed to solve the problem:
https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5403
https://github.com/Koenkk/zigbee2mqtt/issues/9274

if there is anything that can help solve the problem with ZHA integration , i will be happy to contribute if I can.

**Comment by ivalet** [comment]
> Looks like Deconz and Z2M managed to solve the problem: [dresden-elektronik/deconz-rest-plugin#5403](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5403) [Koenkk/zigbee2mqtt#9274](https://github.com/Koenkk/zigbee2mqtt/issues/9274)
> 
> if there is anything that can help solve the problem with ZHA integration , i will be happy to contribute if I can.

Same here!

**Comment by RharoLP** [comment]
On the same boat.

**Comment by KennyStier** [comment]
Will hopefully be resolved by #1151 once merged

**Comment by smiba** [comment]
Just wanted to let you know I have the same issue, have a stack of these sensors but they can't be added.

If there is anything I could do to help please let me know

**Comment by dmulcahey** [comment]
We have them working… unfortunately these devices need the network to reply to a command when joining with a specific manufacturer code. For EZSP coordinators there will be a new bellows release to support them. For TI coordinators you will need a new firmware.

**Comment by RharoLP** [comment]
Branch was merged 7h ago. Has someone tried to add the Aqara E1 Door Sensor again?

**Comment by dmulcahey** [comment]
You need other changes depending on coordinator. TI and Conbee need newest firmware and EZSP needs newest bellows lib. Once that is in place the device will need to be freshly paired.

**Comment by smiba** [comment]
@dmulcahey Not to bother you, but do you know if the latest ConBee II firmware released on the 19th of August 2021 does provide support for what's required? I don't think these sensors were out yet back then which makes me think the ConBee II is just not supported at the moment?

**Comment by dmulcahey** [comment]
No idea. I have personally confirmed the bellows (EZSP) changes and the z2m folks have confirmed and released firmware updates for the TI stuff.

**Comment by RharoLP** [comment]
> EZSP needs newest bellows lib

sorry newbie here, how do I update the bellows lib I can confirm I have an EZSP device?  Should I remove my Zigbee integration and add it again?

**Comment by dmulcahey** [comment]
Just wait for the beta tomorrow

**Comment by RharoLP** [comment]
Hello,
I tried yesterday to add the sensor into HA it didn't recognise it. Do I need to add the Bellows lib this way? [https://github.com/zigpy/bellows#testing-new-releases](url)

**Comment by robtpaton** [comment]
Hi all, I'm running ZHA (Home Assistant 2021.12.4) with a Conbee II (firmware 12.13.4). From my understanding from reading the above, it was expected that at this point, these devices should pair and work properly. However I still get the same behaviour mentioned when this issue was opened. Has anyone else had luck getting them to work with these versions/am I missing something?

TIA

**Comment by RharoLP** [comment]
> 

Hi Rob,
After the December update it worked for me.

**Comment by robtpaton** [comment]
> 

Thanks @RharoLP and this is with a Conbee II? Do you mind posting which versions of things you're running?

**Comment by RharoLP** [comment]
I’m not using Conbee sorry. I’m using a Nortek USB stick. 

From previous posts you need to upgrade your firmware

`https://github.com/zigpy/zha-device-handlers/issues/1088#issuecomment-985129243 `

**Comment by smiba** [comment]
It's working for me and I use a ConBee II

It does still show them with an "on/off" state though (off being closed), but the sensors work and detect changes

**Comment by robtpaton** [comment]
![Screenshot_20211221-193516](https://user-images.githubusercontent.com/1998104/146987563-8ad0098d-3d87-4582-a470-ee074cf87345.jpg)

Interesting @smiba I must be missing something. Screenshot of the device in my HA attached, for me battery always stays unknown and contact always stays off. How does the screenshot compare to yours? Appreciate the help, it feels like I'm so close

**Comment by smiba** [comment]
Mine just has a properly exposed status and battery, I have four of this model and all of them worked right out of the box. 

Not sure what could go wrong, do you have multiple and do they all fail? Or could this one be bad?

Mine for example:

![Screenshot_21_12_2021__20_39](https://user-images.githubusercontent.com/3308142/146988054-61594f8d-a095-4a6b-b850-8479af73f11c.jpg)

**Comment by robtpaton** [comment]
@smiba have tried three of them now, all show the same :(

Yes actually I'm running 12.13.4 which was only released a few days ago, I was under the impression that was needed for these sensors (). I used Phoscon app inside Deconz add-on to update the firmware of the Conbee II, but then stop the Deconz add-on before trying to pair. I'll try and post the logs from when I try to pair one in case that helps

**Comment by dmulcahey** [comment]
If you have the most current firmware you need to fully reset the device and then pair it again. Preexisting pairings will not work

**Comment by RharoLP** [comment]
> It's working for me and I use a ConBee II
> 
> It does still show them with an "on/off" state though (off being closed), but the sensors work and detect changes

@smiba you need to add some lines in your configuration.yaml

homeassistant:
(tab)customize:
(tab)(tab)_deviceid_
(tab)(tab)(tab) device_class: door

That will change to open/close, you can substitute the door in class for any of the folowing [classes](https://www.home-assistant.io/integrations/binary_sensor/
)

**Comment by robtpaton** [comment]
> If you have the most current firmware you need to fully reset the device and then pair it again. Preexisting pairings will not work

Thanks @dmulcahey yes I've tried factory resetting the sensors, tried removing the devices and re-pairing, and 1 of the 3 sensors I tried this with, I'd never even tried to pair before updating the firmware 🤔

**Comment by smiba** [comment]
> @smiba you need to add some lines in your configuration.yaml
> That will change to open/close, you can substitute the door in class for any of the folowing [classes](https://www.home-assistant.io/integrations/binary_sensor/)

@RharoLP Cheers, I ended up doing so in the Developer Settings > State and added `device_class: door `

I think this sensor should be a `device_class: opening` by default though, but I don't know where to submit a PR for this. The old sensors were listed as opening. 

---

> Yes actually I'm running 12.13.4 which was only released a few days ago, I was under the impression that was needed for these sensors. [...]

@robtpaton I can't confirm it working with 12.13.4, I'm still running 12.13.3, although I highly doubt it broke in the most recent update.
Just for verification, which firmware is your ConBee II running?

**Comment by robtpaton** [comment]
@smiba 

> Just for verification, which firmware is your ConBee II running?

ConBee II Firmware 12.3.4 (aka Panko https://github.com/dresden-elektronik/deconz-rest-plugin/releases/tag/v2.13.4)
I used Deconz Add-on 16.11.1 to install that firmware (https://github.com/home-assistant/addons/commit/e7de969655c4a3bc617527be09751268991932a0#diff-71dcbd189f0dd3e12b9170d939f50babb679cd957f0bfff995bb476fbfd98b40)
I'm running Home Assistant 2021.12.4

I've also noticed this comment on another PR https://github.com/zigpy/zha-device-handlers/pull/1151#issuecomment-979661950 so will also follow up in that thread to see if that poster has had any joy

**Comment by smiba** [comment]
> ConBee II Firmware 12.3.4 (aka Panko https://github.com/dresden-elektronik/deconz-rest-plugin/releases/tag/v2.13.4)
> I used Deconz Add-on 16.11.1 to install that firmware ([home-assistant/addons@e7de969#diff-71dcbd189f0dd3e12b9170d939f50babb679cd957f0bfff995bb476fbfd98b40](https://github.com/home-assistant/addons/commit/e7de969655c4a3bc617527be09751268991932a0#diff-71dcbd189f0dd3e12b9170d939f50babb679cd957f0bfff995bb476fbfd98b40))

Now I might be completely wrong as I've only used my ConBee purely with ZHA, but are you sure that is the actual ZigBee IC firmware? 

I had to flash my update with GCFFlasher and the firmware found over at:[ ](http://deconz.dresden-elektronik.de/deconz-firmware/)

In my case I'm running 0x26720700 on my ConBee II

**Comment by robtpaton** [comment]
Thanks @smiba, flashed manually and now works. For anyone else looking back at this - I'd got sidetracked using the deconz add-on, desperately hoping that would work as I thought GCFlasher wasn't usable on OSX which is all I have available to me right now. However GCFlasher is buildable on OSX so was able to use that. Had to use a USB extension lead to flash, but other than that all good. Thanks all for the help
